src/Helpers/Tap/TappedTabBar.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  ActionType,
+  ContextModule,
+  OwnerType,
+  TappedTabBar,
+} from "../../Schema"
+
+export interface TappedTabBarArgs {
+  badge: boolean
+  tab: string
+}
+
+/**
+ * A user taps an icon in the Tab Bar within the iOS app
+ *
+ * badge is true if present, false if not
+ *
+ * tab describes the icon tapped by the user
+ *
+ * @example
+ * ```
+ * tappedTabBar({
+ *   badge: false,
+ *   tab: "sell"
+ * })
+ * ```
+ */
+export const tappedTabBar = ({ badge }: TappedTabBarArgs): TappedTabBar => {
```

Comment:
if i try to add `tab` to the args list here (after `badge`),  I get an error if i supply an explicit value for `tab:` on line 34. Which is correct? The current syntax or... do I add `tab` to the args list on line 28 and change what's inside the `return` brackets?

Reply:
yep you need to add `tab` here and then change line 34
```diff
     context_screen_owner_type: OwnerType.home,
-    tab: "sell",
+    tab,
   }
```